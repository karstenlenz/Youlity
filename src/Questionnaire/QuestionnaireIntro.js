import React from 'react'
import Button from '../common/Button'
import HeadlineUnderline from '../common/HeadlineUnderline'

export default function QuestionnaireIntro({ onButtonClick }) {
  return (
    <>
      <img alt="" src="/img/questionnaire-intro.svg" />
      <HeadlineUnderline>
        <h1>Schritt 2: Fragebogen</h1>
      </HeadlineUnderline>
      <p>
        Durch die Auswahl Ihrer Bedürfnisse haben wir 2 potentielle
        Persönlichkeitsstile identifiziert. <br /> Bitte füllen Sie jeweils
        einen Fragebogen aus, um die Ausprägung des jeweiligen Stils bei Ihnen
        zu testen. Es werden jeweils 8 Ja/Nein-Fragen gestellt. Pro Bogen dauert
        es ca. 3 Minuten. Die Reaktionszeit spielt keine Rolle, Sie können den
        Bogen in Ruhe ausfüllen.
      </p>
      <Button onClick={onButtonClick}>Weiter zum Fragebogen</Button>
    </>
  )
}
