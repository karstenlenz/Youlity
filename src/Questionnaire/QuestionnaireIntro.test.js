import { render } from '@testing-library/react'
import React from 'react'
import QuestionnaireIntro from './QuestionnaireIntro'
import { BrowserRouter as Router } from 'react-router-dom'

describe('The QuestionnaireIntro component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Router>
        <QuestionnaireIntro />
      </Router>
    )
    expect(container).toMatchSnapshot()
  })
})
