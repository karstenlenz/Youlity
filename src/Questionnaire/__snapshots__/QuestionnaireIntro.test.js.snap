// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The QuestionnaireIntro component renders correctly 1`] = `
<div>
  <img
    alt=""
    src="/img/questionnaire-intro.svg"
  />
  <section
    class="HeadlineUnderline__HeadlineContainer-sc-1reeb6v-0 jWFWIm"
  >
    <h1>
      Schritt 2: Fragebogen
    </h1>
    <svg
      class="HeadlineUnderline__UnderlineStyled-sc-1reeb6v-1 hZOZZy"
    >
      underline.svg
    </svg>
  </section>
  <p>
    Durch die Auswahl Ihrer Bedürfnisse haben wir 2 potentielle Persönlichkeitsstile identifiziert. 
    <br />
     Bitte füllen Sie jeweils einen Fragebogen aus, um die Ausprägung des jeweiligen Stils bei Ihnen zu testen. Es werden jeweils 8 Ja/Nein-Fragen gestellt. Pro Bogen dauert es ca. 3 Minuten. Die Reaktionszeit spielt keine Rolle, Sie können den Bogen in Ruhe ausfüllen.
  </p>
  <button
    class="Button__BtnStyled-sc-129yfz5-0 fyEIpZ"
    width="100"
  >
    Weiter zum Fragebogen
  </button>
</div>
`;
